src/coordinator/storage/local/storage.go

```diff
@@ -22,32 +22,35 @@ package local
 
 import (
 	"context"
+	goerrors "errors"
```

Comment:
Is it standard to alias this, or should we leave this as `"errors"`?

Reply:
The problem is that the following packages are all imported:

Reply:
Oops, missed the coordinator/errors package import